### ISSUE TITLE
Add testId / arial-label property

### ISSUE THREAD

**Issue by PauloMello99** [dvtng/react-loading-skeleton#71]
That's a nice library!! But for testing purposes, it would be awesome to have the properties testId and/or aria-label to implement those tests correctly!

**Comment by srmagura** [comment]
Good idea, will add some way to set the test ID. I've created a separate issue for accessibility improvements.
